docs/tutorials/query-language.md

```diff
@@ -53,6 +53,58 @@ let peopleWhoseNameContainsA = realm.objects('Contact').filtered('name CONTAINS[
 let Johns = realm.objects('Contact').filtered('name ==[c] "john"');
 ```
 
+You can do simple wildcard matching with `LIKE` which supports using `?` to match a single character and `*` to match zero or multiple characters.
+
+Example:
+```JS
+// Matches "John" and "Johnny"
+let Johns = realm.objects('Contact').filtered('name LIKE "John*"');
+```
+
 ### Composition
 Use parentheses and the `&&`/`AND` and `||`/`OR` operators to compose queries. You can negate a predicate with `!`/`NOT`.
 
+### Queries on collections
+
+When objects contain lists you can query into them using the collection operators `ANY`, `ALL` and `NONE`.
+
+Example:
+```JS
+// Find contacts with one or more teenage friends
+let teens = realm.objects('Contact').filtered('ANY friends.age < 14');
```

Comment:
@nirinchev There's an example of a query operating on a list of numbers is here, if it's not clear that the string operators are supported, we could add something like `ANY friends.name CONTAINS[c] "rob"`?

Reply:
I was thinking of an example for a query against a list of strings, e.g. `'ANY keywords CONTAINS[c] "apple"'` but @astigsen mentioned these are not supported :( If that's the case probably we should mention it explicitly?

Reply:
Sadly, querying lists of primitives is not yet supported :(